### ISSUE TITLE
Not working with SublimeLinter 4

### ISSUE THREAD

**Issue by carlosfrodriguez** [oschwald/SublimeLinter-perlcritic#6]
Hello I get the following error with the latest update of SublimeLinter 4

```
SublimeLinter: ERROR: Linter crashed.

Traceback (most recent call last):
  File "lint.backend in /Users/Carlos/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package", line 97, in execute_lint_task
  File "lint.linter in /Users/Carlos/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package", line 794, in lint
  File "lint.linter in /Users/Carlos/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter.sublime-package", line 500, in get_cmd
  File "/Users/Carlos/Library/Application Support/Sublime Text 3/Packages/SublimeLinter-contrib-perlcritic/linter.py", line 30, in cmd
    config = util.find_file(
AttributeError: 'module' object has no attribute 'find_file'
```

**Comment by carlosfrodriguez** [comment]
I've just read that the function util.find_file() has been removed in 
https://github.com/SublimeLinter/SublimeLinter/commit/4838a046608ab39843ee4dd2493ce2d6b8e1b16a#diff-6e6fb941c4b3a249f53ac9a3ef3da85d

Currently I comment the lines as I don't need an specific rc file, but it would be great if you update the package with the new way to load config files :-) than you in advance.

**Comment by oschwald** [comment]
I am not currently using this linter, but I would be happy to accept a PR to fix the issue.

**Comment by jonasbn** [comment]
I have to do some releases of some Perl distributions next week and have tonne of $work, but if it is not a in a hurry I can have a look at it.

**Comment by oschwald** [comment]
This was fixed in #7.
